code/game/objects/structures/crates_lockers/closets/cardboardbox.dm

```diff
@@ -62,11 +66,10 @@
 
 	COOLDOWN_START(src, alert_cooldown, time_between_alerts)
 
-	for(var/mob/living/alerted_mob in alerted)
-		if(alerted_mob.stat == CONSCIOUS)
-			if(!alerted_mob.incapacitated(IGNORE_RESTRAINTS))
-				alerted_mob.face_atom(src)
-			alerted_mob.do_alert_animation()
+	for(var/mob/living/alerted_mob as anything in alerted)
+		if(!alerted_mob.incapacitated(IGNORE_RESTRAINTS))
+			alerted_mob.face_atom(src)
+		alerted_mob.do_alert_animation()
 
 	playsound(loc, 'sound/machines/chime.ogg', 50, FALSE, -5)
```

Comment:
Play the sound only to people that can see it for consistency

Reply:
I don't think it should, I find it funnier that you can visibly hear people being alerted.

Reply:
Ehhhhhhhhhhhhhhhhhhhhhhhhhhhhh... Whatever it's been three weeks